lib/validations.js

```diff
@@ -776,7 +776,7 @@ function ErrorCodes(messages) {
  *   }
  *
  *   // This line shows how to create a ValidationError
- *   err = new ValidationError(model);
+ *   var err = new MyModel.ValidationError(model);
```

Comment:
I believe this should be `var err = new ValidationError(model);` for example see how it is used in other places as well: https://github.com/strongloop/loopback-datasource-juggler/blob/5bf18b07280e22d713e722b8e23f8aee14c6587e/lib/relation-definition.js#L2195